src/DFTK.jl

```diff
@@ -10,6 +10,9 @@ using LinearAlgebra
 using Interpolations
 using Requires
 
+using TimerOutputs
+const timer = TimerOutput()
```

Comment:
Should we have a macro that uses this? Eg `@timer begin ... end` which would expand to `@timeit timer begin ... end`?

Reply:
Pros: Less typing, Cons: What if we want a second kind of timer?
I think since this rarely happens an `@timer` macro sounds like a good idea.

Reply:
Yes, I think it would be better too.